types/ssh2/index.d.ts

```diff
@@ -697,6 +697,11 @@ export class Client extends EventEmitter {
      * the server.
      */
     openssh_forwardOutStreamLocal(socketPath: string, cb: ClientCallback): this;
+
+    /**
+     * Calls setNoDelay() on the underlying socket. Disabling Nagle's algorithm improves latency at the expense of lower throughput.
+     */
+    setNoDelay(noDelay: boolean): this;
```

Comment:
The docs seem to imply `noDelay` is optional.

Reply:
You're correct - I missed that. `setNoDelay` in `ssh2` is a proxy for Node's [`net.Socket#setNoDelay`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/net.d.ts#L199), where the `noDelay` argument is optional

Reply:
Gotcha, added this commit to address that
[fix: make noDelay optional to match the doc](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/72329/commits/a2f4c5e81d05f5183dbad1c753ce3e6096fe3810)